csharp-advanced-topics/DeprecationInCsharp/DeprecationTests/DeprecationTest.cs

```diff
@@ -1,29 +1,29 @@
-using Deprecation;
+using Deprecation.StringUtils;
```

Comment:
This can go in the global usings.

Reply:
using Deprecation.StringUtils; moving it to global usings not working